pkgs/test_reflective_loader/lib/test_reflective_loader.dart

```diff
@@ -151,10 +151,22 @@ void _addTestsIfTopLevelSuite() {
   if (_currentSuiteLevel == 0) {
     void runTests({required bool allGroups, required bool allTests}) {
       for (var group in _currentGroups) {
+        var runningTestCount = 0;
         if (allGroups || group.isSolo) {
           for (var test in group.tests) {
             if (allTests || test.isSolo) {
-              test_package.test(test.name, test.function,
+              test_package.test(test.name, () async {
+                runningTestCount += 1;
```

Comment:
Hm... Lets increment before `test_package.test`, not during running it.
Otherwise we will do 0 -> 1 -> 0 (tear off) -> 1 -> 0 (tear off 2) ...